### DISCUSSION TITLE
Line Break in reports

### DISCUSSION THREAD

**Discussion by fletchersmum** [sheltermanager/asm3#1594] [Q&A / Help]
I hope I can make it clear what I am asking here!

I have information in the Notes sections for animals, Comments, Hidden comments etc. When I key them in, I can enter and have a line break, but when I create a report, the lines run into each other. Is there a way to get line breaks?

I can see in the manuals to put AS [fieldname]n2br. No matter what I do I can't get this to work. Not sure if it just doesn't work, or I am clueless as to where to put it and not formatting it properly!

TIA

**Comment by Tetzor** [comment]
Hi, in the SQL editor the field should be given an alias with n2br at the end.

Before:
animal.AnimalComments, 
After:
animal.AnimalComments AS AnimalCommentsn2br, 

In the HTML editor you would also then need to update the HTML for this field with the alias:  

Before:
$AnimalComments  
After :
$AnimalCommentsn2br

Jon

**Reply by fletchersmum** [reply]
Thanks for that. I hadn't realised I had to put it in both SQL and HTML! Works now.

**Comment by Nika250** [comment]
Thank you !
